rust/noosphere/src/ffi/mod.rs

```diff
@@ -0,0 +1,14 @@
+mod key;
```

Comment:
Consider: module level docs here indicating all(?) FFI representations must live in here